tests/testthat.R

```diff
@@ -1,3 +1,3 @@
 library(testthat)
 library(hdm)
-#test_check("hdm")
```

Comment:
Note that I activated unit tests here. In the current master they are commented out.